responsive-web-design-piano/css/style53.css

```diff
@@ -0,0 +1,89 @@
+/* box model reset */
+html {
+    box-sizing: border-box;
+}
+
+
+*, *::before, *::after {
+    box-sizing: inherit;
+}
+
+
+/* piano styles */
+#piano {
+    width: 992px;
+    height: 290px;
+    margin: 80px auto;
+    padding: 90px 20px 0 20px;
+    background-color: #00471b;
+    position: relative;
+    border-radius: 10px;
+}
+
+
+.keys {
+    width: 949px;
+    height: 180px;
+    padding-left: 2px;
+    background-color: #040404;
+}
+
+
+.key {
+    position: relative;
+    width: 41px;
+    height: 175px;
+    margin: 2px;
+    float: left;
+    background-color: #ffffff;
+    border-radius: 0 0 3px 3px;
+}
+
+
+.logo {
+    width: 200px;
+    position: absolute;
+    top: 23px;
+}
+
+
+/* pseudo elements */
+.key.black--key::after {
+    content: "";
+    position: absolute;
+    left: -18px;
+    width: 32px;
+    height: 100px;
+    background-color: #1d1e22;
+    border-radius: 0 0 3px 3px;
+}
+
+
+/* media queries */
+
+/* devices smaller then 768px */
+@media only screen and (max-width: 768px) {
+
+    #piano {
+        width: 355px;
+    }
+
+    .keys {
+        width: 318px;
+    }
+
+    .logo {
+        width: 150px;
+    }
+
+}
+
+
+/* 
+    You might have noticed the keys collapses when the piano's
```

Comment:
You might have noticed the keys collapse when the browser window
    is smaller than 768px. To take care of this issue, set `overflow` to `hidden`.
    in the `.keys` rule. This property will hide any element that is pushed outside
    the set `width` of `.keys`.